brix-core/src/main/java/org/brixcms/plugin/site/SitePlugin.java

```diff
@@ -649,6 +650,12 @@ public SiteTab(IModel<String> title, IModel<Workspace> workspaceModel) {
         public Panel newPanel(String panelId, IModel<Workspace> workspaceModel) {
             return new NodeManagerContainerPanel(panelId, workspaceModel);
         }
+        
+        @Override
+        public boolean isVisible() {
```

Comment:
For ITab it is fine. Since it is not a Component its `isVisible()` is called exactly once per rendering.